src/actions/base.ts

```diff
@@ -95,7 +97,7 @@ export class BaseAction {
     const isSingleNumber: RegExp = /^[0-9]$/;
     const isSingleAlpha: RegExp = /^[a-zA-Z]$/;
 
-    for (let i = 0, j = 0; i < one.length; i++, j++) {
+    for (let i = 0, j = 0; i < one.length; i++ , j++) {
```

Comment:
I think prettier will handle this - can you run `gulp forceprettier` please?

Reply:
Fixed!